calico_containers/pycalico/netns.py

```diff
@@ -180,6 +179,32 @@ def remove_ip_from_ns_veth(container_pid, ip, veth_name_ns):
                        "dev", "%(device)s" % veth_name_ns])
 
 
+def provision_veth(self, endpoint, ns_pid, veth_name_ns):
```

Comment:
Note, that change is somewhat unrelated to this PR, so I'm happy to make that change myself if you want to merge this as is.